README.md

```diff
@@ -4,7 +4,7 @@
 [![Code Coverage](https://codecov.io/gh/repobee/repobee/branch/master/graph/badge.svg)](https://codecov.io/gh/repobee/repobee)
 [![Documentation Status](https://readthedocs.org/projects/repobee/badge/?version=stable)](http://repobee.readthedocs.io/en/stable/)
 [![PyPi Version](https://badge.fury.io/py/repobee.svg)](https://badge.fury.io/py/repobee)
-![Supported Python Versions](https://img.shields.io/badge/python-3.7%2C%203.8-blue.svg)
+![Supported Python Versions](https://img.shields.io/badge/python-3.7%2C%203.8%2C%203.10-blue.svg)
```

Comment:
3.9 is not here even though we run tests for it. Should I add it in a separate PR?

Reply:
Add it now.